src/oc_erchef/apps/oc_chef_wm/src/oc_chef_wm_base.erl

```diff
@@ -84,7 +84,29 @@ service_available(Req, #base_state{reqid_header_name = HeaderName} = State) ->
             State1 = State0#base_state{server_api_version = Version},
             State2 = set_req_contexts(Req, State1),
             State3 = maybe_with_default_org(Req, State2),
-            {true, Req, State3}
+            case wrq:path(Req) of
+                "/_status" ->
+                    {true, Req, State3};
+                _ ->
+                    case chef_license_worker:get_license() of
```

Comment:
can we handle this case statement as a function, something like 
```
_ -> 
  check_license(chef_license_worker:get_license(), Req, State3) 
```

With: 
``` 
check_license({valid_license..... }, Req, State ) ->
  { true, Req, State} ;
check_license({commercial_grace_period...
... etc
```

Makes it a  little easier to test, and avoids deeply nesting the contents of `service_available` which makes it easier to parse.